### PR TITLE
Add UMAP with concat normalized data from plates 5, 3, and 3 prime

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Add UMAP with concat normalized data from plates 5, 3, and 3 prime

In this PR, normalized data from plates 5, 3, and 3 prime were concat together to generate UMAP embeddings and visualized using cell count, genotype, and plate number. 

Please feel free to provide suggestions on the plots that will be used and considered for the manuscript.

**NOTE:** The larger code line charges are due to the UMAP embedding files.

### CODE REVIEW

File: 4.analyze_data/notebooks/UMAP/figures/UMAP_Concat_plate.png
Comment (MikeLippincott) [inline 4.analyze_data/notebooks/UMAP/figures/UMAP_Concat_plate.png:1]: Nice! Across the plates too, this looks good!

File: 4.analyze_data/notebooks/UMAP/scripts/1.vis_UMAP.r [hunk 1]
```diff
@@ -139,11 +141,46 @@ umap_siRNA_construct_gg <- (
...
+    ggplot(df, aes(x = UMAP0, y = UMAP1))
+    + geom_point(
+            aes(color = Metadata_Plate), size = 1.2, alpha = 0.5
+    )
+    + theme_bw()
+    + scale_color_manual(
+        name = "Plate",
+        values = c("Plate_3" = "#7570b3", "Plate_3_prime" = "#e7298a", "Plate_5" = "#d95f02")
+    )
+)
```
Comment (MikeLippincott) [inline 4.analyze_data/notebooks/UMAP/scripts/1.vis_UMAP.r:183]: Note this is a stylistic comment and feel free to ignore as this is a personal preference.

I tend to write my R code more pythonically. See example below and feel free to accept or reject!

```suggestion
umap_plate_gg <- (
    ggplot(
    	df, 
    	aes(
    		x = UMAP0, 
    		y = UMAP1
		)
	)
    + geom_point(
        aes(
	       	color = Metadata_Plate
        ), 
        size = 1.2, alpha = 0.5
    )
    + theme_bw()
    + scale_color_manual(
        name = "Plate",
        values = c(
        	"Plate_3" = "#7570b3", 
        	"Plate_3_prime" = "#e7298a", 
        	"Plate_5" = "#d95f02"
    	)
    )
)
```

**Review by MikeLippincott — APPROVED** [review]
LGTM. Great normalization.

File: 4.analyze_data/notebooks/UMAP/figures/UMAP_Concat_plate.png
Comment (jenna-tomkinson) [reply 4.analyze_data/notebooks/UMAP/figures/UMAP_Concat_plate.png:1]: Ty! 🎉

File: 4.analyze_data/notebooks/UMAP/scripts/1.vis_UMAP.r (hunk 1 above)
Comment (jenna-tomkinson) [reply 4.analyze_data/notebooks/UMAP/scripts/1.vis_UMAP.r:183]: This is a fair comment! Do you happen to know of a linter that already does this or do you have to manually do this as you write the R code?

**Comment by jenna-tomkinson** [comment]
@MikeLippincott Thank you for the review! I am merging now!

**Pull request merged**